bazel/oci/install_packages.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+apt-get update
+
+export DEBIAN_FRONTEND="noninteractive"
+export TZ="Etc/UTC"
+
+apt-get install --yes \
+    build-essential \
+    curl \
+    git \
+    openjdk-8-jdk \
```

Comment:
Should we use OpenJDK 11 now? Bazel itself at least no longer supports JDK 8 (although I think you can still use it to build stuff).